pkg/schema/type.go

```diff
@@ -26,6 +26,8 @@ type Type interface {
 	SetTitle(string)
 	GetExamples() []Example
 	SetExamples([]Example)
+	IsDeprecated() (bool, string)
+	SetDeprecated(deprecated bool, notice string)
```

Comment:
```suggestion
	SetDeprecated(bool, string)
```

We can remove the names here